### DISCUSSION TITLE
Keyball61の特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by frisk2020** [Yowkees/keyball#497] [Q&A]
![Picsart_24-03-04_22-08-02-779](https://github.com/Yowkees/keyball/assets/162134842/baf5bfce-a07c-417d-9d74-74d9bc1c6268)
画像の赤丸の二カ所について、キーが反応しない状態になっています。
ロープロ部分は通常のものも半田付けしていますが、どちらも反応しない状態です。
ソケットの付け直しをしてみたりしていますが解決せず、アドバイス頂けますと幸いです。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

反応しないキーに対応しているダイオードがそれぞれの色どうしのものになりますので、ダイオードと根気よくキーソケットのはんだ付けのやり直しをお願いいたします。
ダイオードのはんだ付けがまるまるとしており、うまくパッドの部分に接続できていない可能性がありますので、はんだの量を調節しながら作業をしていただけますと幸いです。

![309763800-baf5bfce-a07c-417d-9d74-74d9bc1c6268](https://github.com/Yowkees/keyball/assets/143698349/15a2abb5-d9e2-4b7c-b050-abcd7a52a977)
